### PR TITLE
Fixed paste key in IE11

### PR DESCRIPTION
**Pull request by hiro-su**
hello, i always use this project.
can't get data ```ev.clipboardData``` in IE11.
this pr will fix it, so please check.

### CODE REVIEW

File: src/handlers/Clipboard.js [hunk 1]
```diff
@@ -50,11 +50,24 @@ function copyHandler (ev) {
  */
 function pasteHandler(ev, term) {
   ev.stopPropagation();
-  if (ev.clipboardData) {
-    var text = ev.clipboardData.getData('text/plain');
+
+  var paste = function() {
```
Comment (parisk) [inline src/handlers/Clipboard.js]: 1. Please add a `text` argument to the function.
2. I would prefer it if the function was renamed to something more semantic like `dispatchPaste`

File: src/handlers/Clipboard.js [hunk 2]
```diff
@@ -50,11 +50,24 @@ function copyHandler (ev) {
...
     term.handler(text);
     term.textarea.value = '';
     return term.cancel(ev);
+  };
+
+  var userAgent = window.navigator.userAgent.toLowerCase();
+  if (userAgent.match(/msie|MSIE/) || userAgent.match(/(T|t)rident/)) {
+    if (window.clipboardData) {
+      var text = window.clipboardData.getData('Text');
+      paste();
```
Comment (parisk) [inline src/handlers/Clipboard.js]: Please call this as `paste(text)`.

File: src/handlers/Clipboard.js [hunk 3]
```diff
@@ -50,11 +50,24 @@ function copyHandler (ev) {
...
+    }
+  } else {
+    if (ev.clipboardData) {
+      var text = ev.clipboardData.getData('text/plain');
+      paste();
```
Comment (parisk) [inline src/handlers/Clipboard.js]: Please call this as `paste(text)`.

**Comment by parisk** [comment]
Thanks for this @hiro-su! Just a few code comments to be fixed and I think it's good to go.

**Comment by hiro-su** [comment]
@parisk Thank you for review. I made request changes. Can you be able to check them?

**Comment by parisk** [comment]
Great @hiro-su, thanks!

**Comment by parisk** [comment]
@hiro-su just letting you know that we mentioned you for this contribution (along with #339 and #345), in the [**2.1** announcement](http://xtermjs.org/blog/articles/2016-11-10-xtermjs-2.1). Thank you!

**Pull request merged**